src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseEditableBlock.tsx

```diff
@@ -71,6 +71,7 @@ const ReleaseEditableBlock = ({
           editable={editable && !isBrowser('IE')}
           id={blockId}
           label=""
+          hideLabel
```

Comment:
Good to add this prop :+1: 
Can you put the label text back as this will still be rendering an empty span if you give it an empty string and the label might be useful for accessibility?